convex/createAttendee.ts

```diff
@@ -1,23 +1,36 @@
-import { Id } from "convex/values";
 import authenticatedMutation from "./helpers/authenticatedMutation";
+import { Id } from "./_generated/dataModel";
 
 export type CreateAttendeeInput = {
-  eventId: Id;
+  eventId: Id<"events">;
 };
 
 export default authenticatedMutation(
   async ({ db }, user, { eventId }: CreateAttendeeInput) => {
+    const event = await db.get(eventId);
+    if (!event) {
```

Comment:
This pattern is showing up a lot, where we expect a query to succeed but it's possible that no results are found. I'm not sure if there's a better practice here for returning a useful error message to the client.

Reply:
This came up in Discord the other day too, cc @ianmacartney for advocating for the dev experience here. In this case what about adding a ! so that this is a runtime error instead of silently returning `null`? Agreed this seems common enough that I should be able to ask for an id unconditionally if I have reason to believe it will be there.

Reply:
If we switch `db.get` to throw an error if the object doesn't exist, should we make a separate method to attempt to get an object that returns `null`? `db.getOrNull`?

But yeah, I agree that this is gross and we should do something about it.

Reply:
`getOrNull` is nice!